src/main/java/com/mazawrath/beanbot/commands/beanmarket/BeanInvestCommand.java

```diff
@@ -34,18 +35,30 @@ public void onCommand(String[] args, ServerTextChannel serverTextChannel, User a
                     "\t- `.beaninvest buy [symbol] [amount]` - Buys shares from that symbol.\n" +
                     "\t- `.beaninvest sell [symbol]` - Sells all shares bought from that symbol.");
         } else if (args[0].equals("buy")) {
+            EmbedBuilder embed = new EmbedBuilder()
+                    .setTitle("Bean Market Investment")
+                    .setThumbnail("https://cdn.discordapp.com/attachments/489203676863397889/503334187621941308/Stock.png");
             if (args.length >= 3) {
                 if (stockMarket.isProperDecimal(args[2])) {
                     if (StockMarket.getSymbol(args[1].toUpperCase()) != null) {
                         if (new BigDecimal(args[2]).setScale(Points.SCALE, Points.ROUNDING_MODE).divide(stockMarket.getStockPrice(StockMarket.getSymbol(args[1].toUpperCase()), false), 2, RoundingMode.HALF_UP).setScale(Points.SCALE, Points.ROUNDING_MODE).compareTo(new BigDecimal("0.01").setScale(Points.SCALE, Points.ROUNDING_MODE)) >= 0) {
```

Comment:
Oh yeah, this if statement has to be changed to use `roundedSharesBought` instead